.github/workflows/playwright.yml

```diff
@@ -0,0 +1,27 @@
+name: Playwright Tests
```

Comment:
this is cool. Will this enable these tests to run on the PRs? How will it know which tests to run?

Reply:
Playwright can recognize its own tests and has integration with GitHub actions/events so we can definitely have it run the tests in response to things like opening or committing to a PR.